.gitignore

```diff
@@ -1,6 +1,8 @@
+.DS_Store
```

Comment:
In general, if possible try to use white lists over black lists. For `.gitignore` you could do something like this:

```
# Dot files
.*

# Git files
!/.gitignore
!/.gitmodules

# Logs files
logs
*.log

# Runtime data
pids
*.pid
*.seed

# NPM dependency directory
node_modules

# Config files
config/*
!config/default.js

# Env files
env/*
!/env/.env.default

# Coverage directory
coverage
```